dev/PushNotifications/PushNotificationUtility.h

```diff
@@ -158,8 +158,14 @@ namespace winrt::Microsoft::Windows::PushNotifications::Helpers
     inline std::wstring GetExeNameFromPath(std::wstring const& exePath)
     {
         size_t pos{ exePath.rfind(L"\\") };
-        THROW_HR_IF(E_UNEXPECTED, pos == std::wstring::npos);
-        return exePath.substr(pos + 1); // One after the delimiter
+        if (pos != std::wstring::npos)
```

Comment:
Why not

```
return std::filesystem::path(exePath).filename();
```

Reply:
The main reason is because I forgot that there is a std::filesystem library...